JuliaInterface/src/JuliaInterface.c

```diff
@@ -900,35 +888,6 @@ static Obj Func_JuliaSetGAPFuncAsJuliaObjFunc(Obj self,
     return NULL;
 }
 
-static Obj FuncJuliaSetAsJuliaPointer(Obj self, Obj obj)
```

Comment:
I would guess this function does nothing anymore anyway, so it could be savely removed